monkestation/code/modules/mob/living/carbon/carbon_death.dm

```diff
@@ -5,7 +5,9 @@
 	return ..()
 
 /mob/living/carbon/revive(full_heal_flags, excess_healing, force_grab_ghost)
+	var/old_stat = stat
 	. = ..()
-	var/policy = get_policy(POLICY_REVIVAL)
-	if(policy)
-		to_chat(src, policy)
+	if(old_stat != DEAD)
```

Comment:
wont this send policy only if you were not dead before due to being !=

Reply:
good catch lol